lib/utils/sendInterviewTimes/formatInterviewEmail.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  emailApplicantInterviewType,
+  emailCommitteeInterviewType,
+} from "../../types/types";
+import { formatDateHours } from "../dateUtils";
+import { changeDisplayName } from "../toString";
+
+export const formatApplicantInterviewEmail = (
+  applicant: emailApplicantInterviewType
+) => {
+  let emailBody = `<p>Hei <strong>${applicant.applicantName}</strong>,</p><p>Her er dine intervjutider for ${applicant.period_name}:</p><ul><br/>`;
+
+  applicant.committees.sort((a, b) => {
+    return (
+      new Date(a.interviewTime.start).getTime() -
+      new Date(b.interviewTime.start).getTime()
+    );
+  });
+
+  applicant.committees.forEach((committee) => {
+    emailBody += `<li><b>Komite:</b> ${changeDisplayName(
+      committee.committeeName
+    )}<br>`;
+    emailBody += `<b>Start:</b> ${formatDateHours(
+      new Date(committee.interviewTime.start)
+    )}<br>`;
```

Comment:
Kunne vært en idé å la `formatDateHours()` gjøre inputet om til Date inni funksjonen, så vi slipper å skrive `new Date(...)` for hver dato?